src/api/auth-api-client.ts

```diff
@@ -98,19 +98,47 @@ export interface TokenLoginForm extends LoginForm {
  * Status code for auth client results
  */
 export enum KnownAuthStatusCode {
-
-  LOGIN_FAILED_REASON = 12,
-  TOO_MANY_TRY_LOGIN = 13,
+  INVALID_PHONE_NUMBER = 1,
+  SUCCESS_WITH_ACCOUNT = 10,
+  SUCCESS_WITH_DEVICE_CHANGED = 11,
+  MISMATCH_PASSWORD = 12,
+  EXCEED_LOGIN_LIMIT = 13,
+  MISMATCH_PHONE_NUMBER = 14,
+  EXCEED_PHONE_NUMBER_CHECK_LIMIT = 15,
+  NOT_EXIST_ACCOUNT = 16,
+  NEED_CHECK_PHONE_NUMBER = 20,
+  NEED_CHECK_QUIZ = 25,
+  DORMANT_ACCOUNT = 26,
+  RESTRICTED_ACCOUNT = 27,
   LOGIN_FAILED = 30,
+  NOT_VERIFIED_EMAIL = 31,
   MOBILE_UNREGISTERED = 32,
+  UNKNOWN_PHONE_NUMBER = 99,
+  SUCCESS_SAME_USER = 100,
+  SUCCESS_SAME_USER_BY_MIGRATION = 101,
+  TOO_MANY_REQUEST_A_DAY = -20,
+  TOO_MANY_REQUEST_AT_A_TIME = -30,
+  MISMATCH_PASSCODE = -31,
+  EXCEED_DAILY_REQUEST_LIMIT = -32,
+  EXCEED_DAILY_REQUEST_LIMIT_VOICECALL = -33,
+  EXCEED_DAILY_REQUEST_LIMIT_WITHOUT_TOKEN = -34,
   DEVICE_NOT_REGISTERED = -100,
   ANOTHER_LOGON = -101,
   DEVICE_REGISTER_FAILED = -102,
   INVALID_DEVICE_REGISTER = -110,
-  INCORRECT_PASSCODE = -111,
```

Comment:
Breaking changes?

Reply:
> BC Break?
> @JellyBrick 

Not a major change, but need to unify the words with other status codes.

Reply:
@JellyBrick 
or how about make two for legacy support?

`INCORRECT_PASSCODE = -111,
INVALID_PASSCODE = -111,`